token-lending/cli/src/main.rs

```diff
@@ -1168,8 +1168,8 @@ fn command_update_reserve(
     if pyth_price_pubkey.is_some() {
         println!(
             "Updating pyth oracle pubkey from {} to {}",
-            reserve.liquidity.pyth_oracle_pubkey.to_string(),
-            pyth_price_pubkey.unwrap().to_string(),
+            reserve.liquidity.pyth_oracle_pubkey,
```

Comment:
why though?

Reply:
clippy didn't like it
smth smth `This crap implements Display, you don't need to_string()`